src/components/Category/Category.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <div class="container">
+    <header class="d-flex justify-content-between">
+      <h2 class="title fw-bold">Danh mục</h2>
+      <button type="button" class="btn btn-success" @click="btnAdd">Thêm danh mục</button>
+    </header>
+    <div class="d-flex align-items-center w-100 mt-2">
+      <form class="d-flex w-75">
+        <input
+          class="form-control me-2 w-75"
+          type="search"
+          placeholder="Tên danh mục"
+          aria-label="Tên danh mục"
+          v-model.lazy.trim="categoryName"
+        />
+      </form>
+    </div>
+    <div>
+      <category-list v-if="filterCategories.length > 0" :categories="filterCategories" @delete-item="deleteBtn" @modify-item="modifyBtn" />
+      <div v-else class="no-content w-50 mx-auto mt-3">
+        <img class="img-fluid" src="../../assets/images/noContent.svg" alt="No Content" />
+      </div>
+    </div>
+    <div v-show="componentName === 'category-add-form'" class="position-absolute popup">
+      <category-add-form @close-popup="closePopup" @add-item="addItem"></category-add-form>
```

Comment:
```suggestion
      <category-add-form @closePopup="closePopup" @addItem="addItem"></category-add-form>
```
props truyền xuống thì đặt kiểu kecab-case còn event truyền xuống thì đặt kiểu camelCase cho anh nhé